bcda/api/requests.go

```diff
@@ -447,6 +447,7 @@ func (h *Handler) bulkRequest(w http.ResponseWriter, r *http.Request, reqType se
 	resourceTypes := h.getResourceTypes(rp, ad.CMSID)
 
 	if err = h.validateResources(resourceTypes, ad.CMSID); err != nil {
+		log.API.Error(err)
```

Comment:
that's a good point! Because there are similar logging statements elsewhere and this will (hopefully) only arise during smoke tests, I think the current line will probably be OK for now. I do often favor verbosity/explicit error messages or something to give us context quickly though.